packages/react-paypal-js/.nvmrc

```diff
@@ -1 +1 @@
-v16
+v18
```

Comment:
oh you know what? we should remove this and just rely on an `.nvmrc` at the root of the project.

Good idea updating this though